src/gui/LayoutMapper.h

```diff
@@ -99,33 +96,32 @@ class LayoutMapper {
      *
      * @return Page index to put at coordinates
      */
-    std::optional<size_t> map(size_t col, size_t row) const;
 
-private:
+public:
     struct internal_data {
         size_t cols = 0;
         size_t rows = 0;
         size_t actualPages = 0;
 
-        int offset = 0;
+        uint32_t offset = 0;
```

Comment:
Now `getFirstPageOffset()` should have return type `uint32_t` as well or be removed (it isn't used anywhere).

Reply:
Now it doesn't compile any more. Maybe some missing include (#include <cstdint> I guess)?
```  
./src/gui/LayoutMapper.h:99:9: error: 'uint32_t' does not name a type; did you mean 'wint_t'?
   99 |         uint32_t offset = 0;
```